price-estimator/src/main.rs

```diff
@@ -158,7 +158,14 @@ fn main() {
         options.orderbook_update_interval,
     ));
 
-    let filter = filter::all(orderbook, token_info).with(warp::log("price_estimator"));
+    // We add the allow origin header so that requests from the interactive openapi documentation
+    // go through to locally running instance.
```

Comment:
Not a fan of this comment as it could be made more general: the statement is true for any CORS request, not just specific to OpenAPI. This also means that the instructions for testing locally will no longer require starting chrome with security enabled. In production and staging, we just have CORS configured in nginx for our deployment, so we never had to configure it at an app level.